seatunnel-engine/seatunnel-engine-server/src/test/java/org/apache/seatunnel/engine/server/CoordinatorServiceTest.java

```diff
@@ -93,8 +95,8 @@ public void testMasterNodeActive() {
                                 CoordinatorService coordinatorService =
                                         server2.getCoordinatorService();
                                 Assertions.assertTrue(coordinatorService.isCoordinatorActive());
-                            } catch (SeaTunnelEngineException e) {
-                                Assertions.assertTrue(false);
+                            } catch (Exception e) {
+                                Assertions.assertInstanceOf(SeaTunnelEngineException.class, e);
```

Comment:
Before this change, it should not throw Exception, why it will throw exception now?